### PR TITLE
feat: add custom time to object metadata

### PR DESCRIPTION
**Pull request by frankyn**
Adding a new metadata field to print out.

### CODE REVIEW

File: storage/src/object_metadata.php [hunk 1]
```diff
@@ -85,6 +85,9 @@ function object_metadata($bucketName, $objectName)
     if (isset($info['retentionExpirationTime'])) {
         printf("retentionExpirationTime: " . $info['retentionExpirationTime'] . PHP_EOL);
     }
+    if (isset($info['customTime'])) {
+        printf("customTime: " . $info['customTime'] . PHP_EOL);
```
Comment (tmatsuo) [inline storage/src/object_metadata.php:89]: It's using `printf` so maybe:

```PHP
        printf("customTime: %s" . PHP_EOL, $info['customTime']);
```
?

**Review by tmatsuo — APPROVED** [review]
I think the added code is consistent with the rest, but I feel we should use the place holder if we use `printf`.

File: storage/src/object_metadata.php (hunk 1 above)
Comment (frankyn) [reply storage/src/object_metadata.php:89]: Good point, I'll send an update.

**Review by tmatsuo — APPROVED** [review]

**Comment by frankyn** [comment]
Thanks @tmatsuo!

**Pull request merged**